<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USReconGUI</class>
 <widget class="QWidget" name="USReconGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_ImageLabel">
       <property name="text">
        <string>ultrasound source:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_ImageComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_TrackingLabel">
       <property name="text">
        <string>tracking source:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_TrackingComboBox"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="m_HorizontalLayout3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="m_NumberOfFramesLabel">
         <property name="text">
          <string>number of frames:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_NumberOfFramesValueLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="m_HorizontalLayout2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="m_GrabSingleFramePushButton">
         <property name="text">
          <string>grab</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_ClearDataPushButton">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ball size (pixels):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="m_BallSize">
         <property name="toolTip">
          <string>Ball size in pixels</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="m_CalibrationLabel">
       <property name="text">
        <string>processing:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="m_CalibrationRunPushButton">
         <property name="text">
          <string>calibrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_ReconstructVolumePushButton">
         <property name="text">
          <string>reconstruct</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="m_ScalingMatrixLabel">
       <property name="text">
        <string>scaling matrix:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QmitkMatrixWidget" name="m_ScalingMatrix" native="true"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="m_RigidMatrixLabel">
       <property name="text">
        <string>rigid matrix:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QmitkMatrixWidget" name="m_RigidMatrix" native="true"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="m_SaveMatchedDataLabel">
       <property name="text">
        <string>save paired data:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="m_SaveMatchedDataPushButton">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>QmitkMatrixWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
